### PR TITLE
Week 5 Assignment Issue

### PR DESCRIPTION
**Pull request by kamibena**
I was trying to create a pdf along with the html file this week and ran into an issue.  Appears my computer is missing a file.  The error message I received started off with "! LaTeX Error: File `booktabs.sty' not found."

I am having trouble figuring out where to find that file and/or getting my pdf knit to work properly.  I would very much appreciate any advice you have for this!

In the meantime, I have included the html file in the pull request.

### CODE REVIEW

**Comment by kamibena** [comment]
I've added week 6 assignment - for some reason, my knit to pdf worked fine for this assignment but still give me issues for week 5.

**Comment by wrshoemaker** [comment]
@kamibena could you resolve these conflicts and resubmit?

File: Week5-Temporal/temporal_assignment.Rmd [hunk 1]
```diff
@@ -49,18 +49,32 @@ rm(list=ls())
...
+require(nlme)
+require(forecast)
+require(lsmeans)
+require("vegan")
+require("ade4")
+require("BiodiversityR")
+require("gplots")
+require("viridis")
+require("indicspecies")
 ```
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: Good observations on question one about species richness!

Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: More description on _how_ diversity varied among sites for question 2 would have been good

File: Week5-Temporal/temporal_assignment.Rmd [hunk 2]
```diff
@@ -318,29 +332,23 @@ pander(anova(rich.rm))
 
 lsmeans(rich.rm, ~plot_type)
 
-#unstructured
-rich.rm <- lme(richness ~ plot_type * year, random = ~ 1 | plot_id,
-               correlation = corSymm(form = ~ 1 | plot_id),
-               data = rich.treat)
-summary(rich.rm)
-anova(rich.rm)
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: Question was to do RM-ANOVA on inverse of Simpson, not richness

File: Week5-Temporal/temporal_assignment.Rmd [hunk 3]
```diff
@@ -353,7 +361,7 @@ d. Of the three RM-ANOVA models with different covariance structures, which one
...
 > ***Answer 4b***:  Based on eyeballing the plot, it seems as if there might be a slight increase of richness over time and the two plot types seems to follow the same general trends (however this may be due to environmental changes).
 > ***Answer 4c***:  Plot_type has a significant effect on rodent richness (F = 12.28, df = 1/12, p < 0.01).  Neither year (F = 0.01, df = 1/343, p = 0.93) nor the interaction between year and plot_type (F = 0.45, df = 1/343, p = 0.51) have a significant effect on rodent richness.  Therefore, there is a significant difference in richness over time when you compare full rodent exclosure from a plot and full rodent access to a plot, and this is most fully explained by the plot_type treatment.  In general, based on the F values treatment means only considerably differ between plot types. 
-> ***Answer 4d***: The autoregressive model worked the best.  This would say that the time intercept is a good measure of diversity change over time.
+> ***Answer 4d***: The autoregressive model appeared to work the best.  This would say that the time intercept is a good measure of diversity change over time.
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: What criteria did you use to figure out which model appeared to work best? AIC? 

Intercept is sort of a nuisance in RM-ANOVA designs, or perhaps better stated, it's a random effects. We account for this by giving each experiment unit (plot) a random intercept. This allows us to calculate errors appropriately.

Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: For question 7, what about: "Test for a biodiversity-stability relationship by regressing community stability on mean inverse Simpson’s diversity."?

Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: Good thoughts on synthesis!

**Pull request merged**